test/scripts/common.inc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Include file with common parts shared for local and ephemeral
+#
+
+function error {
```

Comment:
Since it's a Bash script, not a POSIX sh script, use `()` to define functions:

```suggestion
error() {
```

Reply:
I don't see anything against that in the documentation at: https://www.gnu.org/software/bash/manual/bash.html#Shell-Functions ; if I am missing something happy to know.